addons/mail/static/src/components/chatter/chatter.xml

```diff
@@ -14,9 +14,9 @@
                         t-att-class="{ 'o-bordered': chatter.hasExternalBorder }"
                         composerLocalId="chatter.thread.composer.localId"
                         hasMentionSuggestionsBelowPosition="true"
-                        hasTextInputSendOnEnterEnabled="false"
                         isCompact="false"
                         isExpandable="true"
+                        textInputSendShortcuts="['ctrl-enter', 'meta-enter']"
```

Comment:
I think we'd prefer a chatter test instead (or in complement) of composer tests for these shortcuts.